runtime/themes/papercolor-light.toml

```diff
@@ -0,0 +1,94 @@
+# Palette based on https://github.com/NLKNguyen/papercolor-theme
+# Author: Soc Virnyl Estela <socvirnyl.estela@gmail.com>
+
+"ui.background" = {bg="background"}
+"ui.text" = "foreground"
+"ui.selection" = {bg="selection_background", fg="selection_foreground"}
+"ui.statusline" = {bg="paper_bar_bg", fg="regular0"}
+"ui.statusline.inactive" = {bg="background", fg="bright2"}
+"ui.virtual" = "indent"
+"ui.virtual.whitespace" = { fg = "regular5" }
+"ui.cursor.match" = {bg = "foreground", fg = "regular4"}
+"ui.cursor" = {bg = "foreground", fg = "background"}
+"ui.window" = {bg = "background", fg = "bright3"}
+"ui.menu.selected" = {bg = "selection_background"}
+
+"markup.heading" = { fg = "bright2", modifiers = ["bold"] }
+"markup.list" = "regular4"
+"markup.bold" = { fg = "foreground", modifiers = ["bold"] }
+"markup.italic" = { modifiers = ["italic"] }
+"markup.link.url" = { fg = "regular4", modifiers = ["underlined"] }
+"markup.link.text" = "bright2"
+"markup.link.label" = { fg = "regular7", modifiers = ["bold"] }
+"markup.raw" = "foreground"
+
+"string" = "foreground"
+"attribute" = "bright7"
+"keyword" = { fg = "regular4", modifiers = ["bold"]}
+"keyword.directive" = "regular1"
+"namespace" = "regular1"
+"type" = "bright2"
+"type.builtin" = { fg = "regular4", modifiers = ["bold"]}
+"variable" = "foreground"
+"variable.builtin" = "cyan"
+"variable.other.member" = "regular4"
+"variable.parameter" = "foreground"
+
+"special" = "#3E999F"
+"function"  = "bright1"
+"constructor" = "bright1"
+"function.builtin" = { fg = "regular4", modifiers = ["bold"]}
+"function.macro" = { fg = "regular1" }
+"comment" = { fg = "bright0", modifiers = ["dim"] }
+"module" = "#af0000"
+"constant" = "#5f8700"
+"constant.builtin" = "#5f8700"
+"constant.numeric" = "#d75f00"
+"constant.character.escape" = { fg = "#8700af", modifiers = ["bold"]}
+"operator" = { fg = "regular4", modifiers = ["bold"]}
+
+"label" = { fg = "selection_background", modifiers = ["bold", "italic"] }
+
```

Comment:
I believe it is also missing `diff` colors, but either way I will merge for now.